### ISSUE TITLE
Restructure buildpack: Update language family order groups

### ISSUE THREAD

**Issue by sophiewigmore** [paketo-buildpacks/php#483]
## Context
Per [PHP RFC 0001](https://github.com/paketo-buildpacks/rfcs/blob/main/text/php/0001-restructure.md) we are restructuring the PHP language family of buildpacks to make them more modular, up-to-date, and as a result, maintainable.

## Issue
This issue is to update the language family order groupings to make use of newly added PHP buildpacks:
* `php-fpm` 
* `php-httpd`
* `php-nginx`
* `php-builtin-server`
* `php-memcached-session-handler` (`optional = true`)
* `php-redis-session-handler` (`optional = true`)

These buildpacks will be split into three order groupings, and will take the place of the `php-web buildpack`

**This issue is blocked on issues related to the implementation each of those buildpacks**:
https://github.com/paketo-buildpacks/php/issues/476
paketo-buildpacks/php-httpd#2
https://github.com/paketo-buildpacks/php/issues/479
https://github.com/paketo-buildpacks/php/issues/477
paketo-buildpacks/php-memcached-session-handler#1
paketo-buildpacks/php-redis-session-handler#1

- [ ] Add new order groups, remove PHP web
- [ ] Add new integration tests (and fixtures if needed)
- [ ] Update README accordingly

## Acceptance
1. With the new language family buildpack, I can successfully:
* Build an image to run HTTPD web server with php
* Build an image to run Nginx web server with php
* Build an image to run Built-in php web server
* Build an image to run FPM1 process manager
* Build an image to run a PHP CLI script using Procfile.
* Optionally use composer as application level package manager*
* Optionally use memcached/redis as session handler

2. The order groupings in the PHP Language family buildpack look like: 
```
[[order]] # HTTPD web server

  [[order.group]]
    id = "paketo-buildpacks/php-dist"
    version = ""

  [[order.group]]
    id = "paketo-buildpacks/composer"
    version = ""
    optional = true

  [[order.group]]
    id = "paketo-buildpacks/composer-install"
    version = ""
    optional = true

  [[order.group]]
    id = "paketo-buildpacks/httpd"
    version = ""

  [[order.group]]
    id = "paketo-buildpacks/php-fpm"
    version = ""

  [[order.group]]
    id = "paketo-buildpacks/php-httpd"
    version = ""

  [[order.group]]
    id = "paketo-buildpacks/php-memcached-session-handler"
    version = ""
    optional = true

  [[order.group]]
    id = "paketo-buildpacks/php-redis-session-handler"
    version = ""
    optional = true

  [[order.group]]
    id = "paketo-buildpacks/procfile"
    version = ""
    optional = true

[[order]] # Nginx web server

  [[order.group]]
    id = "paketo-buildpacks/php-dist"
    version = ""

  [[order.group]]
    id = "paketo-buildpacks/composer"
    version = ""
    optional = true

  [[order.group]]
    id = "paketo-buildpacks/composer-install"
    version = ""
    optional = true

  [[order.group]]
    id = "paketo-buildpacks/nginx"
    version = ""

  [[order.group]]
    id = "paketo-buildpacks/php-fpm"
    version = ""

  [[order.group]]
    id = "paketo-buildpacks/php-nginx"
    version = ""

  [[order.group]]
    id = "paketo-buildpacks/php-memcached-session-handler"
    version = ""
    optional = true

  [[order.group]]
    id = "paketo-buildpacks/php-redis-session-handler"
    version = ""
    optional = true

  [[order.group]]
    id = "paketo-buildpacks/procfile"
    version = ""
    optional = true

[[order]] # Built-in web server (web), FPM (php-fpm) & Script (Procfile)

  [[order.group]]
    id = "paketo-buildpacks/php-dist"
    version = ""

  [[order.group]]
    id = "paketo-buildpacks/composer"
    version = ""
    optional = true

  [[order.group]]
    id = "paketo-buildpacks/composer-install"
    version = ""
    optional = true

  [[order.group]]
    id = "paketo-buildpacks/php-fpm"
    version = ""

  [[order.group]]
    id = "paketo-buildpacks/php-builtin-server"
    version = ""

  [[order.group]]
    id = "paketo-buildpacks/php-memcached-session-handler"
    version = ""
    optional = true

  [[order.group]]
    id = "paketo-buildpacks/php-redis-session-handler"
    version = ""
    optional = true

  [[order.group]]
    id = "paketo-buildpacks/procfile"
    version = ""
    optional = true
```

**Comment by sophiewigmore** [comment]
Closing in favour of #524
